.github/workflows/node-ci.yml

```diff
@@ -140,11 +146,43 @@ jobs:
             - name: Jest with coverage
               run: yarn coverage
 
+            - name: Upload Artifact
+              uses: actions/upload-artifact@v2
+              with:
+                  name: jest-coverage
+                  path: ./coverage/coverage-final.json
+
+    upload_coverage:
+        name: Upload Coverage
+        runs-on: ubuntu-latest
+        needs: [cypress, coverage]
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Use Node.js ${{ matrix.node-version }} & Install & cache node_modules
+              uses: Khan/actions@shared-node-cache-v0
+              with:
+                  node-version: ${{ matrix.node-version }}
+
+            - name: Download Jest Coverage
+              uses: actions/download-artifact@v2
+              with:
+                  name: jest-coverage
+                  # path to decompress the artifact into
+                  path: ./
+
+            - name: Download Cypress Coverage
+              uses: actions/download-artifact@v2
+              with:
+                  name: cypress-coverage
+                  # path to decompress the artifact into
+                  path: ./
```

Comment:
Can you leave a comment explaining how we handle these coverage files? I'm confused why we're downloading artifacts (well, I have a guess but a comment would clarify)?  Do previous jobs upload the coverage artifacts to Github and then this process downloads them all and pushes them to Codecov?

Reply:
The only way to share files between jobs is using artifacts.  I'll leave a comment about this.